modules/sagemaker/sagemaker-groundtruth/README.md

```diff
@@ -0,0 +1,87 @@
+## Introduction
+This module provides an implementation for leveraging Amazon SageMaker Ground Truth for annotating images with missing labels using bounding boxes.
+
+
+## Description
+
+The module orchestrates the labeling process using an AWS Step Functions workflow. The high-level steps involved are:
+
+- Retrieve a list of raw images from S3 and check Amazon SageMaker Feature Store for images with missing labels.
+- Initiate a labeling job in SageMaker Ground Truth to label the remaining images.
+- After the labeling job is completed, kick off a label verification job in SageMaker Ground Truth.
+- Write the verified labels to the Feature Store.
+
+Each of these steps is implemented using AWS Lambda functions. Simple wait/status check loops are used to monitor the status of the Ground Truth jobs.
+## Architecture
+![SageMaker Ground Truth Labelling Architecture](docs/_static/Sagemaker-groundtruth-module-Architecture.png "SageMaker Ground Truth Labelling Architecture")
+
+
+
+## Deployment
+The deployment of the required assets (Lambda functions, IAM roles, etc.) for the Step Functions workflow is automated using an AWS CDK app. The pipeline is triggered on a schedule as well as on Git commits, using two pipelines in AWS CodePipeline. The architecture of the CI/CD infrastructure deployed by the CDK app is as follows:
+
+
+## Inputs/Outputs
+
+### Input Parameters
+#### Required
+- Repository Details:
+
+  - assets bucket: The name of S3 bucket with assets for the lambda functions and images for labelling.
+  - repoName: The name of the repository where the source code is stored.
+  - branchName: The branch name to use from the repository.
```

Comment:
I think the modules usually use kebab-case for parameters in the documentation (`branch-name`) and then snake-case in the code (`branch_name`).